add-package.sh

```diff
@@ -1,8 +1,15 @@
 #!/usr/bin/env bash
 set -euo pipefail
 test -n "$1" || { echo "You must specify a package name. If you want a specific version, append @VERSION"; exit 1; }
+test -n "$2" || { echo "You must specify a the repo(s) to use, or NULL for default behaviour."; exit 1; }
 
 export PACKAGE="$1"
+if [ "$2" = "NULL" ]; then
```

Comment:
I think we should just specify the default value of repos as NULL

e.g. remove line 4, and replace 7-9 with:

```
REPOS=${2:-NULL}
```

And rather than add escape quotes in bash, I would just add them in the R fragment, as per suggested change. Then we're consistantly hanlding package and repo the same way.

Reply:
It's great to add a default but I suspect that the original non-escaping of the quotes around `$REPOS` is the better way to go because in R `NULL` is null, whereas `"NULL"` is not null, i.e.

``` r
is.null(NULL)
#> [1] TRUE
is.null("NULL")
#> [1] FALSE
```